sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -232,11 +233,15 @@ def submit_helper(self, call, logpath, rqmt, name, task_name, start_id, end_id,
         :param int step_size:
         """
         name = self.process_task_name(name)
-        sbatch_call = ["sbatch", "-J", name, "-o", logpath + "/%x.%A.%a", "--mail-type=None"]
+        out_log_file = f"{logpath}/%x.%A.%a"
+        if rqmt.get("multi_node_slots", 1) > 1:
+            out_log_file += ".%t"
+        sbatch_call = ["sbatch", "-J", name, "--mail-type=None"]
         sbatch_call += self.options(rqmt)
+        sbatch_call += ["-o", f"{out_log_file}.batch"]
```

Comment:
What is written to this log file? So we have two log files now?

Reply:
The log file is mostly empty. E.g.:

```
--------------------- Slurm Task Prolog ------------------------
Job ID: 4078734
Job name: i6_core.returnn.training.ReturnnTrainingJob.6iUi8jNHCHUX.run
Host: g-11
Date: Sat Nov  2 11:09:11 EDT 2024
User: mgunz
Slurm account: science
Slurm partition: gpu-24g
Work dir:
------------------
Node usage:
JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
4078734_1   gpu-24g i6_core.    mgunz  R       0:00      1 g-11
------------------
Show launch script with:
sacct -B -j
------------------
--------------------- Slurm Task Prolog ------------------------
```

I think we should keep it as sisyphus cleans up the engine directory anyway, and if the `srun` call fails for some reason I think the logs of that would end up in the `.batch` log file. Otherwise there's also the option to output to `/dev/null` but then we definitely lose any potentially helpful information when things go wrong.

Reply:
>as sisyphus cleans up the engine directory anyway

does it do that? I thought that the folder is always kept..

Anyway, I agree that the logic is fine